common/workunit/workunit.cpp

```diff
@@ -3027,6 +3028,7 @@ class CWorkUnitFactory : public CInterface, implements IWorkUnitFactory, impleme
                 sortElements(iter, sortOrder.get(), NULL, NULL, unknownAttributes, elements);
                 return conn.getClear();
             }
+            virtual unsigned getTotalElements() { throw MakeStringException(-1, "not implemented"); }
```

Comment:
Yes, the pager doesn't see the postfilter.. so the total as far it is concerned here is the elements.ordinality() and then  adjust when postfiltering